doc/database/hierarchy.md

```diff
@@ -92,14 +92,42 @@ Either two parent binomials, separated by a "x" or "×" or a given binomial, wit
 (see [Wikipedia](https://en.wikipedia.org/wiki/Hybrid_name)).
 So the name does not necessarily say if a plant is a hybrid.
 
-### Rendering
+### Rendering the Unique Name
 
-The unique name must be rendered:
+The unique name of a plant needs to be displayed in the following manner:
 
-- Latin name in _italics_.
-- Cultivar name normal font in single quotes (as in database).
+- Latin name should be displayed in _italics_.
+- The cultivar name should be in regular font, enclosed in single quotes (as stored in the database).
 
-E.g.: _Brassica oleracea italica_ 'Ramoso calabrese'
+For example: _Brassica oleracea italica_ 'Ramoso calabrese'
+
+### Rendering the Complete Name
+
+The complete name of a plant should be displayed as follows:
+
+- If there is a common name, the common name appears first and should start with a capital letter (e.g., "tomato" becomes "Tomato").
+- If it is a plant name from the seed database, the name continues with hyphen `-` and the additional name.
+- This is followed by the unique name in brackets, unique name as described above in `Rendering the Unique Name`.
+
+The general format is:
```

Comment:
Again nothing that you did wrong.
We sometimes only rely on the common name (e.g. in the plant label feature that was just merged).
It would be useful if there was a short paragraph explaining when shortened names are appropriate.

(Also maybe not as part of this PR, but as a follow-up.)

Reply:
I added that based on Markus proposal.